apm-test/e2e/e2e-base/src/main/docker/Dockerfile

```diff
@@ -0,0 +1,33 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+FROM openjdk:8-jre-alpine
+
+LABEL maintainer="kezhenxu94@apache.org"
+
+EXPOSE 8080 11800 12800
+
+ENV SW_HOME=/skywalking
+
+VOLUME $SW_HOME
+
+WORKDIR $SW_HOME
+
+ADD apache-skywalking-apm-bin.tar.gz .
```

Comment:
Don't package the image every time. The better way to mount a volume including the target tar file, you could mapping this volume to any place you like.